internal/infrastructure/mongo/mongodoc/asset_test.go

```diff
@@ -0,0 +1,157 @@
+package mongodoc
+
+import (
+	"github.com/reearth/reearth-backend/pkg/asset"
+	"github.com/reearth/reearth-backend/pkg/id"
+	"github.com/stretchr/testify/assert"
+	"go.mongodb.org/mongo-driver/bson"
+	"testing"
+	"time"
```

Comment:
```suggestion
	"testing"
	"time"

	"go.mongodb.org/mongo-driver/bson"
	"github.com/reearth/reearth-backend/pkg/asset"
	"github.com/reearth/reearth-backend/pkg/id"
	"github.com/stretchr/testify/assert"
```

Are you using `gofmt` and `goimports` to format codes? CI requires codes pass [golangci-lint](https://github.com/golangci/golangci-lint). VSCoders use gopls (one of popular language server implementation) and its formatter (it uses gofmt, goimports, and so on internally).

It would be perfect if your code editor could be set up so that formatting is applied each time the file is saved.